kili/cli.py

```diff
@@ -108,13 +118,27 @@ def create_project(api_key: str,
             --input-type TEXT \\
             --title "Invoice annotation project"
         ```
-
+        ```
```

Comment:
we should add a comment to say that it will create a new project with the same json_interface.
It can be ambiguous wether it also add the assets and the members to...